src/Microsoft.DotNet.Build.CloudTestTasks/PackageFiles/xunitrunner-perf/csvjsonconvertor.py

```diff
@@ -0,0 +1,195 @@
+import csv
+import getopt
+import json
+import os
+import os.path
+import serialobj
+import sys
+sys.path.append(os.getenv('HELIX_SCRIPT_ROOT'))
+import helix.logs
+
+log = helix.logs.get_logger()
+
+def readCSV(csvFile):
+    log.info('Reading '+csvFile)
+    if not os.path.exists(csvFile):
+        raise Exception(csvFile+' could not be found for conversion')
+
+    csvdict = dict()
+    with open(csvFile, 'rb') as csvfile:
+        reader = csv.reader(csvfile)
+        for row in reader:
+            if row[0] not in csvdict:
+                csvdict[row[0]] = dict()
+            else:
+                if row[1] not in csvdict.get(row[0]):
+                    csvdict.get(row[0])[row[1]] = dict()
+                else:
+                    if row[2] not in csvdict.get(row[0]).get(row[1]):
+                        csvdict.get(row[0]).get(row[1])[row[2]] = list()
+                    else:
+                        csvdict.get(row[0]).get(row[1]).get(row[2]).append(row[3])
+
+    return csvdict
+
+def generateJson(opts, csvdict):
```

Comment:
This naming scheme doesn't match other helix naming schemes. it should be generate_json, and read_csv